Marii frontend b

Maryamh12 opened the pull request.

Maryamh12 committed:
my first changes in frontend

Maryamh12 committed:
Change the name of folder to Frontend

Joshf225 reviewed (commented):
great worki marii, but as you can see you have 5000 files you are trying to push to merge with the base which is not needed, so before you merge please add the /node_modules folder to your .gitignore iif you have if not just create one in your root directory, Thanks!

Maryamh12 commented:
Thank you! I've already added the node_modules/ folder to .gitignore and removed it from tracking. The unnecessary files won't be included in the merge. Let me know if there's anything else needed!
